source/_integrations/tts.markdown

```diff
@@ -30,7 +30,8 @@ Depending on your setup, you might need to set a external URL (`external_url`) i
 
 </div>
 
-The following optional parameters can be used with any platform. However, the TTS integration will only look for global settings under the configuration of the first configured platform:
+The following optional parameters can be used with any platform.
+**Note**: When configuring multiple tts platform entries, the TTS integration will only look for global settings under the configuration of the first configured platform entry:
```

Comment:
Notes are handled differently, see:

<https://developers.home-assistant.io/docs/documenting/create-page#html>

Additionally, TTS should be is capitalized.